public/res/css/jstree-themes/default-dark/style.css

```diff
@@ -0,0 +1,1112 @@
+}
+.jstree-default-dark > .jstree-wholerow-ul .jstree-hovered,
+.jstree-default-dark > .jstree-wholerow-ul .jstree-clicked {
+  background: transparent;
+  box-shadow: none;
+  border-radius: 0;
+}
+.jstree-default-dark .jstree-wholerow {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.jstree-default-dark .jstree-wholerow-hovered {
+  background: #555555;
+}
+.jstree-default-dark .jstree-wholerow-clicked {
+  background: #5fa2db;
+  background: -webkit-linear-gradient(top, #5fa2db 0%, #5fa2db 100%);
+  background: linear-gradient(to bottom, #5fa2db 0%, #5fa2db 100%);
+}
+.jstree-default-dark .jstree-node {
+  min-height: 24px;
+  line-height: 24px;
+  margin-left: 24px;
+  min-width: 24px;
+}
+.jstree-default-dark .jstree-anchor {
+  line-height: 24px;
+  height: 24px;
+}
+.jstree-default-dark .jstree-icon {
+  width: 24px;
+  height: 24px;
+  line-height: 24px;
+}
+.jstree-default-dark .jstree-icon:empty {
+  width: 24px;
+  height: 24px;
+  line-height: 24px;
+}
+.jstree-default-dark.jstree-rtl .jstree-node {
+  margin-right: 24px;
+}
+.jstree-default-dark .jstree-wholerow {
+  height: 24px;
+}
+.jstree-default-dark .jstree-node,
+.jstree-default-dark .jstree-icon {
+  background-image: url("32px.png");
+}
+.jstree-default-dark .jstree-node {
+  background-position: -292px -4px;
+  background-repeat: repeat-y;
+}
+.jstree-default-dark .jstree-last {
+  background: transparent;
+}
+.jstree-default-dark .jstree-open > .jstree-ocl {
+  background-position: -132px -4px;
+}
+.jstree-default-dark .jstree-closed > .jstree-ocl {
+  background-position: -100px -4px;
+}
+.jstree-default-dark .jstree-leaf > .jstree-ocl {
+  background-position: -68px -4px;
+}
+.jstree-default-dark .jstree-themeicon {
+  background-position: -260px -4px;
+}
+.jstree-default-dark > .jstree-no-dots .jstree-node,
+.jstree-default-dark > .jstree-no-dots .jstree-leaf > .jstree-ocl {
+  background: transparent;
+}
+.jstree-default-dark > .jstree-no-dots .jstree-open > .jstree-ocl {
+  background-position: -36px -4px;
+}
+.jstree-default-dark > .jstree-no-dots .jstree-closed > .jstree-ocl {
+  background-position: -4px -4px;
+}
+.jstree-default-dark .jstree-disabled {
+  background: transparent;
+}
+.jstree-default-dark .jstree-disabled.jstree-hovered {
+  background: transparent;
+}
+.jstree-default-dark .jstree-disabled.jstree-clicked {
+  background: #efefef;
+}
+.jstree-default-dark .jstree-checkbox {
+  background-position: -164px -4px;
+}
+.jstree-default-dark .jstree-checkbox:hover {
+  background-position: -164px -36px;
+}
+.jstree-default-dark.jstree-checkbox-selection .jstree-clicked > .jstree-checkbox,
+.jstree-default-dark .jstree-checked > .jstree-checkbox {
+  background-position: -228px -4px;
+}
+.jstree-default-dark.jstree-checkbox-selection .jstree-clicked > .jstree-checkbox:hover,
+.jstree-default-dark .jstree-checked > .jstree-checkbox:hover {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Expected selector ".jstree-default-dark .jstree-checked > .jstree-checkbox:hover" to come before selector ".jstree-default-dark.jstree-checkbox-selection .jstree-clicked > .jstree-checkbox:hover" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=css%3AS4664)